front/src/App.vue

```diff
@@ -57,6 +57,7 @@ export default {
   max-width: 500px;
   margin: auto;
   overflow-x: hidden;
+  box-shadow: 0 0 100px 1px gray;
```

Comment:
이정도 어때요?

```suggestion
  box-shadow: 0 0 40px 2px gray;
```
<img width="739" alt="Screen Shot 2020-08-27 at 10 47 21 AM" src="https://user-images.githubusercontent.com/43027225/91374278-d531e280-e852-11ea-9cbc-2d451e014df7.png">